packages/ui/src/components/Button/__stories__/index.stories.tsx

```diff
@@ -12,6 +12,9 @@ export default {
   ],
   parameters: {
     docs: {
+      deprecated: true,
+      deprecatedReason:
+        'This component is deprecated please use ButtonV2 instead.',
       description: {
```

Comment:
You can add the migration link as you did a migration page:

```suggestion
  parameters: {
    docs: {
      deprecated: true,
      deprecatedReason:
        'This component is deprecated please use ButtonV2 instead.',
      migrationLink: '?path=/docs/migrations-button-v1-to-buttonv2--page'
      description: {
```